Vagrantfile

```diff
@@ -34,12 +34,20 @@ Vagrant.configure(2) do |config|
     controlplane.vm.box = "roboxes/ubuntu2004"
 
     controlplane.vm.network :private_network, ip:"#{k8s_linux_kubelet_nodeip}"
-    controlplane.vm.provider :virtualbox do |vb|
-    controlplane.vm.synced_folder "./sync/shared", "/var/sync/shared"
-    controlplane.vm.synced_folder "./forked", "/var/sync/forked"
-    controlplane.vm.synced_folder "./sync/linux", "/var/sync/linux"
-      vb.memory = linux_ram
```

Comment:
do we need to keep the section for VirtualBox with the linux_ram and linux_cpus?

Reply:
If possible we should support both until we get qemu fully working.